icons/matplotlib/matplotlib-plain-wordmark.svg

```diff
@@ -0,0 +1 @@
57-3.028 7.31-4.549 3.028-7.31 3.028zm0-20.372c-2.68 0-5.2 1.044-7.095 2.939a9.963 9.963 0 0 0-2.939 7.095c0 2.68 1.044 5.2 2.939 7.095a9.969 9.969 0 0 0 7.095 2.939c2.68 0 5.2-1.044 7.095-2.939a9.969 9.969 0 0 0 2.939-7.095c0-2.68-1.044-5.2-2.939-7.095a9.963 9.963 0 0 0-7.095-2.939z"/><path fill="#11557c" d="M79.3 72.263c-4.445 0-8.062-3.617-8.062-8.062 0-4.446 3.617-8.063 8.062-8.063s8.062 3.617 8.062 8.063c0 4.445-3.617 8.062-8.062 8.062zm0-16.009c-4.381 0-7.945 3.564-7.945 7.946 0 4.381 3.564 7.945 7.945 7.945s7.945-3.564 7.945-7.945-3.564-7.946-7.945-7.946z"/><path fill="#11557c" d="M79.3 69.928c-3.158 0-5.728-2.569-5.728-5.728s2.569-5.728 5.728-5.728 5.728 2.57 5.728 5.728-2.57 5.728-5.728 5.728zm0-11.339c-3.094 0-5.61 2.517-5.61 5.611s2.517 5.61 5.61 5.61 5.61-2.517 5.61-5.61-2.516-5.611-5.61-5.611z"/><path fill="#11557c" d="M79.3 67.594c-1.871 0-3.394-1.522-3.394-3.394s1.522-3.394 3.394-3.394 3.394 1.522 3.394 3.394-1.523 3.394-3.394 3.394zm0-6.67c-1.807 0-3.276 1.47-3.276 3.276s1.47 3.276 3.276 3.276 3.276-1.47 3.276-3.276-1.47-3.276-3.276-3.276z"/><path fill="#11557c" d="M79.3 65.426c-.676 0-1.226-.55-1.226-1.226s.55-1.226 1.226-1.226 1.226.55 1.226 1.226-.55 1.226-1.226 1.226zm0-2.334a1.11 1.11 0 0 0 0 2.216 1.11 1.11 0 0 0 0-2.216z"/><path fill="#11557c" d="M81.559 64.233h-2.264a.07.07 0 0 1-.067-.071.07.07 0 0 1 .046-.063l2.153-.699a.07.07 0 0 1 .087.044c.076.233.114.476.114.721 0 .037-.03.068-.069.068z"/><path fill="#11557c" d="M79.296 64.233a.07.07 0 0 1-.065-.093l2.386-6.358a.068.068 0 0 1 .089-.04c.674.252 1.304.61 1.867 1.059.03.024.035.067.011.097l-4.234 5.309a.07.07 0 0 1-.054.026z"/><path fill="#11557c" d="M79.296 64.233a.069.069 0 0 1-.053-.025l-5.803-6.951a.068.068 0 0 1 .008-.096l.001-.001a9.148 9.148 0 0 1 3.817-1.891.07.07 0 0 1 .083.052l2.015 8.827a.069.069 0 0 1-.037.077.065.065 0 0 1-.031.008z"/><path fill="#11557c" d="M71.501 65.648a.069.069 0 0 1-.068-.056 8.012 8.012 0 0 1 .663-4.895.069.069 0 0 1 .092-.032l7.138 3.437a.069.069 0 0 1 .038.071.07.07 0 0 1-.056.059l-7.795 1.415-.012.001z"/><path fill="#11557c" d="m75.575 66.811-.012-.001a.067.067 0 0 1-.044-.029 4.558 4.558 0 0 1-.363-.623.07.07 0 0 1-.003-.053.07.07 0 0 1 .035-.039l4.079-1.964a.071.071 0 0 1 .09.027.07.07 0 0 1-.02.091l-3.721 2.578a.074.074 0 0 1-.041.013z"/><path fill="#11557c" d="M79.298 69.891c-.43 0-.861-.048-1.277-.143a.07.07 0 0 1-.052-.083l1.259-5.517c.007-.032.037-.059.069-.054a.07.07 0 0 1 .066.057l.948 5.579a.07.07 0 0 1-.056.08 5.68 5.68 0 0 1-.957.081z"/><path fill="#11557c" d="M84.941 71.312a.07.07 0 0 1-.054-.026l-5.645-7.079a.068.068 0 0 1 .005-.091.069.069 0 0 1 .091-.006l7.142 5.566a.07.07 0 0 1 .012.097 9.104 9.104 0 0 1-1.508 1.525.076.076 0 0 1-.043.014z"/><path fill="#11557c" d="M79.358 74.43h-.117V64.342l-7.133 7.133-.083-.083.042-.041 7.091-7.092H69.07v-.117h10.089l-7.134-7.134.083-.083 7.133 7.133V53.97h.117v10.089l7.134-7.134.083.083-.042.041-7.092 7.093h10.088v.117H79.442l7.133 7.133-.083.083-.041-.042-7.093-7.092z"/></svg>
```

Comment:
Please merge the paths if possible in accordance to the [SVG Standards](https://github.com/devicons/devicon/wiki/SVG-Standards)